src/Config/Validator/Deploy/DeprecatedVariables.php

```diff
@@ -88,6 +104,15 @@ public function validate(): Validator\ResultInterface
             );
         }
 
+        if (isset($config[DeployInterface::VAR_STATIC_CONTENT_SYMLINK])
+            && $config[DeployInterface::VAR_STATIC_CONTENT_SYMLINK] === false
```

Comment:
why do you check only false value? probably this notification better to add in any cases

Reply:
`$config` comes from `MergedConfig` which includes the variables' default values (`true`), so we are in effect testing if the variable has been changed from its default state.

Reply:
hm. Then we should remove default value and in the src/Filesystem/RecoverableDirectoryList.php just check the false value, otherwise behave like we would have true

Reply:
oh, I see that we will have an exception in this case. Not good